src/cf-expandables/src/ExpandableToggle.js

```diff
@@ -0,0 +1,92 @@
+/* ==========================================================================
+   Toggle Expandable
+   Mixin for toggling the expandable.
+   ========================================================================== */
+
+'use strict';
+
+var contains = require( 'atomic-component/src/utilities/dom-class-list' ).contains;
+var addClass = require( 'atomic-component/src/utilities/dom-class-list' ).addClass;
+var closest = require( 'atomic-component/src/utilities/dom-closest' ).closest;
+var bind = require( 'atomic-component/src/utilities/function-bind' ).bind;
+var removeClass = require( 'atomic-component/src/utilities/dom-class-list' ).removeClass;
+var ExpandableTransition = require( 'atomic-component/src/utilities/transition/ExpandableTransition' );
+var Events = require( 'atomic-component/src/mixins/Events.js' );
+
+var ToggleExpandable = {
+
+  transition: null,
```

Comment:
Should be set to `UNDEFINED`. Typically only use null when setting variables that will reference dom elements.